### DISCUSSION TITLE
Can I mining partially?

### DISCUSSION THREAD

**Discussion by heeroz** [Chia-Network/chia-blockchain#6604] [Q&A]
I have plotted 50TB. I understand that if I want to join the official pool, all plots need to be replotted. 

My plan is:
・Create a new full node and join hpool with the existing 50TB.
・Create another full node in a different network(avoid UPnP problem) to join the official pool in the future. 
Is this plan possible?

Thank you in advance.

**Comment by loppefaaret** [comment]
sounds solid
